pkg/scheduler/objects/node.go

```diff
@@ -49,6 +50,7 @@ type Node struct {
 	allocations       map[string]*Allocation
 	schedulable       bool
 	unlimited         bool
+	ready             bool
```

Comment:
On this abstraction level, this field could have a different name like `acceptingContainers` or just `healthy`.

Reply:
I think so too, isn't it up to us to pick one?

Reply:
Checking the shim side and K8s itself I think we need to stick with _Ready_ to not confuse things.